app/src/main/java/com/medmapper/v33001/MainActivity.kt

```diff
@@ -1,43 +1,47 @@
 package com.medmapper.v33001
 
 import android.os.Bundle
-import androidx.activity.ComponentActivity
-import androidx.activity.compose.setContent
-import androidx.compose.foundation.layout.fillMaxSize
-import androidx.compose.material.MaterialTheme
-import androidx.compose.material.Surface
-import androidx.compose.material.Text
-import androidx.compose.runtime.Composable
-import androidx.compose.ui.Modifier
-import androidx.compose.ui.tooling.preview.Preview
-import com.medmapper.v33001.ui.theme.MedMapperTheme
-
-class MainActivity : ComponentActivity() {
+import android.view.Menu
+import android.view.MenuItem
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+
+class MainActivity : AppCompatActivity() {
+
+    private val fragmentManager: FragmentManager = supportFragmentManager
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContent {
-            MedMapperTheme {
-                // A surface container using the 'background' color from the theme
-                Surface(
-                    modifier = Modifier.fillMaxSize(),
-                    color = MaterialTheme.colors.background
-                ) {
-                    Greeting("Android")
-                }
+        setContentView(layoutInflater.inflate(R.layout.main_activity, null))
+        setSupportActionBar(findViewById(R.id.toolbar))
+        setFragment(ScheduleFragment())
```

Comment:
initialize layout

Reply:
There are significant changes to this file.  It looks like you're moving it from Jetpack Compose to Fragments.  What is the tech debt reduction you achieve with this change?